zeppelin-interpreter/src/main/java/org/apache/zeppelin/helium/HeliumPackage.java

```diff
@@ -18,13 +18,16 @@
 
 import org.apache.zeppelin.annotation.Experimental;
 
+import java.util.Map;
+
 /**
  * Helium package definition
  */
 @Experimental
 public class HeliumPackage {
   private HeliumType type;
   private String name;           // user friendly name of this application
+  private String version;
```

Comment:
I see what you mean. 

- using `artifact` as key of config is the simplest way to support local, remote helium packages.
- version may be undefined or the same value which published versions have.

considering both cases, it would be good to use `artifact` as key of config. I will modify this PR.